numpy/_typing/_array_like.py

```diff
@@ -39,7 +39,8 @@
 # any and all remaining overloads
 @runtime_checkable
 class _SupportsArray(Protocol[_DType_co]):
-    def __array__(self) -> ndarray[Any, _DType_co]: ...
+    def __array__(self) -> ndarray[Any, _DType_co]:
+        ...
```

Comment:
I think all these typing fixes should be avoided. @jorenham Thoughts?

Reply:
See https://github.com/numpy/numpy/pull/27308#issuecomment-2332596335

Reply:
What can I say? I'm just enforcing pycodestyle rules, not my choice. Unlike ruff, pycodestyle appears to be very opinionated, in the sense that errors cannot be silenced with `noqa`.

Reply:
`# fmt: skip` maybe?

Reply:
ruff also includes pycodestyle btw: https://docs.astral.sh/ruff/rules/#pycodestyle-e-w